exasol_integration_test_docker_environment/lib/base/ssh_access.py

```diff
@@ -106,6 +106,6 @@ def from_cache(cls, cache_directory: Path = None) -> 'SshKey':
             os.makedirs(cache.directory, mode=0o700, exist_ok=True)
             return (
                 cls.generate()
-                .write_private_key(priv)
-                .write_public_key(cache.public_key)
+                .write_private_key(str(priv))
```

Comment:
Probably the parameters should here have type path if they expect a path, but this is maybe something for another PR